app/assets/stylesheets/common/admin/customize.scss

```diff
@@ -23,6 +23,50 @@
   }
 }
 
+.show-post-edits-user-modal {
+  .modal-header {
+    border-bottom: 0;
+  }
+
+  .modal-inner-container {
+    max-width: 70%;
+    padding: 10px 20px;
+
+    .table {
+      border: 0;
+      width: 100%;
+
+      tr {
+        text-align: left;
+      }
+
+      tr:hover {
+        background-color: var(--primary-very-low);
+      }
+
+      td,
+      th {
+        padding: 8px;
+      }
+
+      @media screen and (max-width: 970px) and (min-width: 768px) {
```

Comment:
Very very minor: I wonder whether you need the rules here to be limited to this small set of viewport sizes. Surely the table heading vertical alignment and the sortable element's max width should be set more generally? 

Also, it feels odd that we're applying less padding for this subset of viewports, maybe `padding: 6px 4px;` is not needed at all?

Reply:
Hey, I copied this style from the admin-report component; I removed this bit, and it made no difference, so let's keep it out for now.